core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -196,11 +212,50 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public boolean hasAvailableSubtitles() {
+        return !exoPlayer.getSubtitleTracks().isEmpty();
+    }
+
+    @Override
+    public void selectSubtitleTrack(PlayerSubtitleTrack subtitleTrack, final PlayerSubtitlesView playerSubtitlesView) {
+        setSubtitleRenderer(playerSubtitlesView);
+        exoPlayer.selectSubtitleTrack(subtitleTrack);
+        playerSubtitlesView.showSubtitles();
+    }
+
+    @Override
+    public void selectFirstAvailableSubtitleTrack(PlayerSubtitlesView playerSubtitlesView) {
+        setSubtitleRenderer(playerSubtitlesView);
+        exoPlayer.selectFirstAvailableSubtitlesTrack();
+        playerSubtitlesView.showSubtitles();
+    }
+
+    private void setSubtitleRenderer(final PlayerSubtitlesView playerSubtitlesView) {
+        exoPlayer.setSubtitleRendererOutput(new TextRenderer.Output() {
```

Comment:
Is it ok to pass the `PlayerSubtitlesView` as a dependency of this methods? we already have an `attach` method and I wonder if this `setSubtitleRenderer` can be done there... 

I can also appreciate that this class do very little direct interaction with `PlayerView` and does stuff on it via listeners, so I wonder if we should be doing the same.